installation/cross_compile/README.md

```diff
@@ -0,0 +1,115 @@
+# Cross-Compilation
+Since processing power of the Raspberry Pi Zero module is insufficient for "quick" compilation we made tools to help you compile ROS2 on your PC for the Raspberry Pi Zero.
```

Comment:
```suggestion
Since the processing power of the Raspberry Pi Zero module is insufficient for "quick" compilation, we made tools to help you compile ROS2 on your PC for the Raspberry Pi Zero.
```